src/slic3r/GUI/ObjectDataViewModel.cpp

```diff
@@ -70,6 +70,7 @@ const std::map<InfoItemType, InfoItemAtributes> INFO_ITEMS{
 //           info_item Type                         info_item Name              info_item BitmapName
             { InfoItemType::CustomSupports,      {L("Paint-on supports"),       "fdm_supports_" },     },
             { InfoItemType::CustomSeam,          {L("Paint-on seam"),           "seam_" },             },
+            { InfoItemType::CustomBrim,          {L("Paint-on brim"),           "brim_" },             },
```

Comment:
I also need help from the Prusa community or the Prusa design team for the `brim_` svg.